server/text_generation_server/models/custom_modeling/flash_qwen2_modeling.py

```diff
@@ -251,7 +235,8 @@ def forward(
         max_s,
         prefill_cache_indices,
     ):
-        normed_hidden_states, res = self.input_layernorm(hidden_states, residual)
+        residual = hidden_states
+        normed_hidden_states, _ = self.input_layernorm(hidden_states)
```

Comment:
```suggestion
        normed_hidden_states, residual = self.input_layernorm(hidden_states)
```

should be the same as long as no residual is passed?

https://github.com/huggingface/text-generation-inference/blob/88fd56f549637b36529582b682c4c76eb78e36a5/server/text_generation_server/layers/layernorm.py#L44

Reply:
oh good catch, i've updated to avoid the extra variable/line and confirmed the outputs match the logits in the transformers implementation